[WC-2406]: Combobox sort displayed items on Multi Select (#974)
mendix/web-widgets: packages/pluggableWidgets/combobox-web/CHANGELOG.md

## [Unreleased]

### Fixed

> -   We fixed an issue with the sorting of the displayed items on Multi Selection comboboxes, where the items inside the input were not being sorted exactly like the items inside the menu.

Reviewer: ```suggestion
-   We fixed an issue with the sorting of the displayed selected items in Multi Selection, where the sorting of selected items did not match the sorting in the menu.
```
